scripts/wait_seconds.js

```diff
@@ -0,0 +1,17 @@
+const { waitForNSeconds } = require("../test/utilities.js")
```

Comment:
We will need to remember to change the `mine_blocks` usages in dex-services to `wait_seconds`